apis/core.oam.dev/v1beta1/componentdefinition_types.go

```diff
@@ -27,6 +27,9 @@ import (
 
 // ComponentDefinitionSpec defines the desired state of ComponentDefinition
 type ComponentDefinitionSpec struct {
+	// +optional
+	Version string `json:"version,omitempty"`
```

Comment:
Does this mean that a cluster can only have one version of a component at a time? What happens when consumers refer to a version that's different from the ComponentDefinition version?

Reply:
We are implementing 1.1 proposal. Please refer to the Acceptance Criteria mentioned in [this KEP](https://github.com/kubevela/kubevela/pull/6600/files?short_path=c6da3da#diff-c6da3da65cafd2d214c7ee0801d9db6ffa1105cb277e45f63bc3ba985209a308). 
Will update the PR to add the KEP and docs as part of the same PR.

Reply:
PR description updated.